src/content/docs/en/guides/assets.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: Assets (Experimental)
+description: Learn how to enable experimental asset support in Astro.
+i18nReady: false
+---
+
+**Built-in Asset Support** is enabled in Astro behind an experimental flag. This built-in feature is intended to replace the optional `@astrojs/image` integration. The new assets experience features:
+
+- Brand new, more intuitive `Image` component
```

Comment:
```suggestion
- A new built-in `Image` component
```
I prefer not to tell the reader that something is intuitive!

Reply:
Yep, great call!